docs/src/pages/company/careers/product-designer.md

```diff
@@ -1,92 +1,74 @@
-# Product Designer
+# Designer
 
 <p class="description">Design is critical to the success of our mission. We are looking for skills that complement our Lead Designer. You will empower our audience that seeks to build outstanding-looking UIs with new tools.</p>
 
-## Details of the Role
+## About the company
 
-- Location: Remote (preference for UTC-6 to UTC+3).
-- Type of work: Full-time (contractor or employee [depending on the case](https://www.notion.so/mui-org/Hiring-FAQ-64763b756ae44c37b47b081f98915501))
-- Start date: Immediately.
-- We're a remote company, we prefer asynchronous communication over meetings.
-- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.
+MUI started with Material UI, the most successful React implementation of Google’s Material Design. It has gotten a massive following not only due to the fidelity to Material but also because its number of components, carefully designed component API, obsession for details, and community engagement. Today, countless teams and organizations rely on our open-source projects to build their design system.
```

Comment:
```suggestion
MUI started with Material-UI, the most successful React implementation of Google’s Material Design. It has gotten a massive following not only due to the fidelity to Material but also because its number of components, carefully designed component API, obsession for details, and community engagement. Today, countless teams and organizations rely on our open-source projects to build their design system.
```